app/helpers/date_helper.rb

```diff
@@ -3,7 +3,7 @@ def local_date(datetime, show_year: true)
     datetime = Time.zone.parse(datetime) if datetime.is_a?(String)
 
     tag.time(
-      datetime.strftime("%b %-e#{', %Y' if show_year}"),
+      I18n.l(datetime, format: "%b %-e#{', %Y' if show_year}"),
```

Comment:
Definitely agree with this. I ran into some issues on the client side assumptions about time with how we localize on the frontend, so for the scope of this PR, I'm going to omit time localization and give it its own follow-up work.

Rather than make this a 130 file PR that gets to everything, I'll avoid anything which requires its own intricacies and instead get this to the point of a solid 80 file PR which gets the ball rolling in the most stable way, and make note of work that needs to follow.

Reply:
Ok, that sounds totally reasonable, thanks for the context.